pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -124,6 +124,17 @@ func (r *RepoSyncReconciler) Reconcile(ctx context.Context, req controllerruntim
 		return controllerruntime.Result{}, status.APIServerError(err, "failed to get RepoSync")
 	}
 
+	if !rs.DeletionTimestamp.IsZero() {
```

Comment:
TODO: Move this reconciler-manager fix to its own PR. No updates should be made by the reconciler-manager after deletion starts until after deletion completes (NotFound), at least until we add the reconciler-manager finalizer.

Reply:
Looks like this may not actually be required. Removing to verify with presubmits.